src/main/java/com/gmail/goosius/siegewar/SiegeWar.java

```diff
@@ -43,17 +43,15 @@ public void onEnable() {
     	
         if (!townyVersionCheck(getTownyVersion())) {
             System.err.println(prefix + "Towny version does not meet required minimum version: " + requiredTownyVersion.toString());
-            System.err.println(prefix + "Shutting down....");
             onDisable();
             return;
         } else {
             System.out.println(prefix + "Towny version " + getTownyVersion() + " found.");
         }
         
-        if (!Settings.loadSettingsAndLang()) {
-        	System.err.println(SiegeWar.prefix + "Shutting down....");
+        if (!Settings.loadSettingsAndLang())
         	onDisable();

```

Comment:
:construction: 
- Hi, although it is not in the current code, should there actually be a return statement here?

Reply:
its pretty redundant yes

Reply:
- Oh, I meant "should" in terms of.........I was thinking there should maybe be a return statement in BOTH of the above "If" blocks
- Thus for example, if there were no return statement, then by my reading, even if the version, config, or lang files are found to be bad, it would continue and turn on the plugin anyway (which would seem to be undesired behaviour...)

Reply:
Ah, no it won't go onward to start the rest of the plugin, onDisable() will end it.